### ISSUE TITLE
Chance to get a MacOS Version

### ISSUE THREAD

**Issue by josepuma** [LeNitrous/sbtw#7]
It's possible to compile a mac os version?

**Comment by elvodqa** [comment]
Yes but no. You can build on macOS just fine and still work with the editor. You just can't open the file dialog when creating a project. You can try to implement it If you want tho (◕‿◕✿)
